app/mailers/commontator/subscriptions_mailer.rb

```diff
@@ -0,0 +1,50 @@
+class Commontator::SubscriptionsMailer < ActionMailer::Base
+  helper Commontator::SharedHelper
+  def comment_created(comment, recipients)
+    setup_variables(comment, recipients)
+
+    mail(@mail_params).tap do |message|
+      message.mailgun_recipient_variables = @mailgun_recipient_variables if @using_mailgun
+    end
+  end
+
+  protected
+
+  def setup_variables(comment, recipients)
+    @comment = comment
+    commentUser = User.find(comment.creator_id).first_name + " " + User.find(comment.creator_id).last_name
```

Comment:
we can probably set a `user` variable here and use `.find(...)` once.
also, we can interpolate instead of concat.
but more importantly, once you set the `user` variable, you can use:
```
user.full_name
```
as seen in `app/models/user.rb` line 103
to get the first name and last name in a preformatted string!
or, you could maybe skip the variable declaration and just do:
```
User.find(comment.creator_id).full_name
```
that would work, too

Reply:
Done - didn't see that full_name property.  good catch